pkg/models/project.go

```diff
@@ -1,16 +1,16 @@
 package models
 
 type CloudProvider struct {
-	CloudProviderId   string `json:"cloudProviderId,omitempty" mapstructure:"cloud_provider_id"`
-	CloudProviderName string `json:"cloudProviderName,omitempty" mapstructure:"cloud_provider_name"`
+	CloudProviderId   string `json:"cloudProviderId,omitempty" tfsdk:"cloud_provider_id"`
+	CloudProviderName string `json:"cloudProviderName,omitempty" tfsdk:"cloud_provider_name"`
 }
 
 type Project struct {
-	ProjectId      string          `json:"projectId,omitempty" mapstructure:"project_id"`
-	ProjectName    string          `json:"projectName,omitempty" mapstructure:"name"`
-	UserCount      int             `json:"userCount,omitempty" mapstructure:"user_count"`
-	ClusterCount   int             `json:"clusterCount,omitempty" mapstructure:"cluster_count"`
-	CloudProviders []CloudProvider `json:"cloudProviders" mapstructure:"cloud_providers"`
+	ProjectId      string           `json:"projectId,omitempty" tfsdk:"project_id"`
+	ProjectName    string           `json:"projectName,omitempty" tfsdk:"project_name"`
+	UserCount      int              `json:"userCount,omitempty" tfsdk:"user_count"`
+	ClusterCount   int              `json:"clusterCount,omitempty" tfsdk:"cluster_count"`
```

Comment:
Here, the linter is confused, I guess. Also, the precommit hook is complaining about this indentation. 
```suggestion
	ProjectId      string          `json:"projectId,omitempty" tfsdk:"project_id"`
	ProjectName    string          `json:"projectName,omitempty" tfsdk:"project_name"`
	UserCount      int             `json:"userCount,omitempty" tfsdk:"user_count"`
	ClusterCount   int             `json:"clusterCount,omitempty" tfsdk:"cluster_count"`
```